delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -144,20 +160,20 @@ node {
 		switch (flowType) {
 			case "API":
 				stage('Undeploy Service') {
-					updateServiceNameConfig(domain, service_name, "dev")
+					updateServiceNameConfig(domain, service_name, "dev",config)
```

Comment:
Shouldn't we update the signature for updateServiceNameConfig to take only config as config will contain domain and service_name too?